test/integ/test_connection.py

```diff
@@ -555,7 +555,7 @@ def test_eu_connection(tmpdir):
     import os
 
     os.environ["SF_OCSP_RESPONSE_CACHE_SERVER_ENABLED"] = "true"
-    with pytest.raises(OperationalError):
+    with pytest.raises((OperationalError, InterfaceError)):
```

Comment:
does it still raise OperationalError?
```suggestion
    with pytest.raises(InterfaceError)):
```